commonTestResources/exampleOpenApiFiles/valid/openapi2DefineBasePath.yml

```diff
@@ -0,0 +1,17 @@
+swagger: '2.0'
+info:
+  description: Has one path.
+  title: Example OpenApi 2 spec with basePath
+  version: 0.1.0
+basePath: /test
+paths:
+  /responseBody/string:
+    get:
+      produces:
+        - application/json
+      parameters: []
+      responses:
+        '200':
```

Comment:
```suggestion
        200:
```

Reply:
Fixed in [this commit](https://github.com/RuntimeTools/OpenAPIValidators/pull/160/commits/efa5fbe049f8b015818cb4412fb8a9235031e15a).